<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeteorSlide2</class>
 <widget class="QDialog" name="MeteorSlide2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ground Impact</string>
  </property>
  <widget class="QPushButton" name="NextButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>490</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QLabel" name="TextLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>331</width>
     <height>281</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="AnimationLabel">
   <property name="geometry">
    <rect>
     <x>529</x>
     <y>69</y>
     <width>600</width>
     <height>500</height>
    </rect>
   </property>
   <property name="text">
    <string>Ground impact animation goes here</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BackButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>490</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="backLabel">
   <property name="geometry">
    <rect>
     <x>529</x>
     <y>69</y>
     <width>600</width>
     <height>500</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>610</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Exctinction</string>
   </property>
  </widget>
  <zorder>backLabel</zorder>
  <zorder>NextButton</zorder>
  <zorder>TextLabel</zorder>
  <zorder>AnimationLabel</zorder>
  <zorder>BackButton</zorder>
  <zorder>startButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
